Documentation/En/Index.xyl

```diff
@@ -26,28 +26,28 @@
   abbreviated SUT, and is composed of two parts:</p>
   <ol>
     <li><strong>Test data</strong> to execute the SUT, and</li>
-    <li>An <strong>oracle</strong>, establishing the <strong>test
+    <li>an <strong>oracle</strong>, establishing the <strong>test
     verdict</strong>: Are the result of the execution and the state of
-    the SUT after its execution the ones we expect or not?</li>
+    the SUT after it's execution the ones we expect or not?</li>
   </ol>
   <p>The values of the verdict are: <strong>Success</strong>,
   <strong>fail</strong> or <strong>inconclusive</strong>. A success means this
   is what we expect, a fail means this is not what we expect and an
-  inconclusive means this is not possible to give an answer. The important
+  inconclusive means it is not possible to give an answer. The important
   part is: <q>“what <em>we</em> expect”</q>. By extension, we understand a
-  test does not check the SUT is <strong>verified</strong>.</p>
+  test does not check that the SUT is <strong>verified</strong>.</p>
   <blockquote cite="Dijkstra">Testing shows the presence, not the absence of
   bugs.</blockquote>
   <p>A test only checks the SUT is <strong>valid</strong>. The difference is
   that a valid system has no bug for some <strong>non exhaustive
   executions</strong>, but a verified system does what it is designed to do.
-  Despite tests do not proof the absence of bugs, they are useful (easy to
+  Despite this, tests do not proove the absence of bugs, they are useful (easy to
```

Comment:
I think `this` is required, otherwise it reads as if `despite` were a type of test. However I think despite could be replaced with `nevertheless` (néanmoins?)